app/assets/javascripts/application.js.erb

```diff
@@ -95,13 +95,10 @@ jQuery(document).ready(function ($) {
   });
 });
 
-function checkAll (id, checked) {
-  var els = Element.descendants(id);
-  for (var i = 0; i < els.length; i++) {
-    if (els[i].disabled === false) {
-      els[i].checked = checked;
-    }
-  }
+function checkAll(selector, checked) {
+  jQuery('#' + selector + ' input:checkbox').each(function() {
+    this.checked = checked;
+  });
```

Comment:
will add the check but it doesn't make any difference as disabled form fields are not submitted anyway.